vision_agent_tools/tools/owlv2.py

```diff
@@ -103,3 +101,6 @@ def __call__(
         if len(inferences) == 0:
             return None
         return inferences
+
+    def to(self, device: Device):
+        self._model.to(device=device.value)
```

Comment:
@hrnn Nice! I think we have more models that we might need to add this method to. Could you please add them? (e.g florencev2, depth_anything_v2, ...)

Reply:
Fixed